src/components/ReportsPage/HeatmapPage/HeatmapPage.tsx

```diff
@@ -56,6 +75,26 @@ export const HeatmapPage = () => {
     return buildUrl(queryParamsObj, location.pathname);
   }, [location, selectedScorecardId, groupBy, headerType]);
 
+  const scorecardsResult = useCortexApi(api => api.getScorecards());
+  const excludedGroupBys = useMemo(() => {
+    const teamBasedCardSelected = scorecardsResult.value?.some(
+      (scorecard) => scorecard.id === selectedScorecardId && scorecard.filter?.type === FilterType.TEAM_FILTER
```

Comment:
@FoodProduct Correct me if I'm wrong, but is it still possible to create a scorecard with filtertype == TEAM_FILTER? Or it is only for old scorecards and now we define them via `COMPOUND_FILTER`. I'm filtering for teams-related scorecards on Teams graph page, so probably you need to do it the same way in here:

https://github.com/cortexapps/brain-app/blob/9530ad64ba2c9782c1f301fc53ce0b73d82b5a7c/apps/%40cortexapps/brain-app/src/components/Pages/Graph/FlowGraph/GraphToolbar/HierarchyOptionsButton.tsx#L53-L56

Reply:
Right, so we need to look for team-based scorecards in the same way it is done on Graph page in brain-app.

Reply:
I had to copy paste the types for this as well, I simplified them as much as i could.
I also added check for the CQL filter. Could be maybe missing in the brain-app condition?